lib/SPIRV/LLVMSaddWithOverflow.h

```diff
@@ -0,0 +1,244 @@
+//===- LLVMSaddWithOverflow.h - implementation of llvm.sadd.with.overflow -===//
+//
+//                     The LLVM/SPIRV Translator
+//
+// This file is distributed under the University of Illinois Open Source
```

Comment:
Since you are adding some new files, should we reconsider the copyright header on those? (see https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/80)

Reply:
I'm not sure that it is okay to have files with different licenses in one project. 
I'd prefer to see clarifications on the Khronos re-licensing process from Neil. I think it will be more consistent if we change all copyright headers in scope of other PR.

Reply:
Fair enough, I'm fine leaving those headers as they are.